extensions/discovery-and-network/services.yaml

```diff
@@ -0,0 +1,247 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: services
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: services
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |-
+    header:
+      - name: Controlled by
+        source: metadata.ownerReferences
+        widget: ControlledBy
+      - name: Type
+        source: spec.type
+      - name: Cluster IP
+        source: spec.clusterIP
+      - name: Ports
+        source: >-
```

Comment:
name of port is not displayed, try to rethink how to make this view cleaner and easier to understand for user, previously we had something like this, I know that currently we couldn't make something like this, but come with new proposal which will display all informations about port
![image](https://user-images.githubusercontent.com/46759954/227471144-ce533f60-2c0c-4e02-8b4c-7e9522159825.png)